packages/Webkul/Shop/src/Resources/views/emails/layout.blade.php

```diff
@@ -4,7 +4,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
 
-        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.googleapis.com" crossorigin>
```

Comment:
Please use multiline where there is more than one attribute. Also please check for any existing case.